src/server/utilities/cspHeader/index.js

```diff
@@ -19,8 +19,10 @@ const directives = {
       'https://a1.api.bbc.co.uk/hit.xiti', // ATI
       'https://toggles.api.bbci.co.uk', // Toggles service
       'https://cdn.ampproject.org', // AMP
+      'https://*.ampproject.net', // Social Embeds
```

Comment:
Needed since this file is requested from this domain: https://3p.ampproject.net/2004030010070/f.js

Reply:
Yes, and this file seems to change 'version' quite a bit.